From 0e1b178cf1d53db22936b7ff7d4482b183fd0edc Mon Sep 17 00:00:00 2001
From: Pino Toscano <ptoscano@redhat.com>
Date: Mon, 13 Oct 2014 11:02:14 +0200
Subject: [PATCH 066/102] edit: fix --format documentation (RHBZ#1151738).

Use only --format=FMT as a way to specify a format, as the version
without the equal sign will not work for that.

(cherry picked from commit 145a188f7056d4414adef472e95cad5640d79511)
---
 edit/virt-edit.pod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/edit/virt-edit.pod b/edit/virt-edit.pod
index 4bc2dce..6387919 100644
--- a/edit/virt-edit.pod
+++ b/edit/virt-edit.pod
@@ -129,7 +129,7 @@ being altered by the shell.
 
 Note that this option is only available when Perl 5 is installed.
 
-=item B<--format> raw|qcow2|...
+=item B<--format=raw|qcow2|..>
 
 =item B<--format>
 
-- 
1.9.3

